### PR TITLE
Added routes

### PR DESCRIPTION
**Pull request by shristabaruah**
Added routes to the pages 
changed product listing page to react

### CODE REVIEW

File: src/pages/Homepage/components/hero/hero.jsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  return (
+    <header className="hero-section">
+      <div className="img-container">
+        <img className="hero-img" src={hero} />
+        <div className="content">
+          <p className="sub-heading">THE BEST SELECTION OF QUALITY DEVICES</p>
+          <h2 className="sub-header">QUALITY ELECTRONICS</h2>
+          <h3>Smart Tech objects that will astound you</h3>
+          <Link to="/products" className="product-link">
+            <button className="btn default-outline shop-btn">SHOP NOW</button>
```
Comment (gadgetfather) [inline src/pages/Homepage/components/hero/hero.jsx:15]: You can wrap your content in Link tag and style it like a button.

**Review by gadgetfather — APPROVED** [review]
- well written code
- you can add _htmlFor_ tags in label
- paths are working fine 👍

**Comment by shristabaruah** [comment]
> * well written code
> * you can add _htmlFor_ tags in label
> * paths are working fine 👍

Thanks  @gadgetfather Learnt something new will use _htmlFor_ tags from now on.

**Review by AtulPant2704 — APPROVED** [review]
Everything looks fine, the is well structured, just one advice which Akshay has already mentioned above to replace `for` attribute in the label to `htmlFor`, rest all looks good.

**Comment by shristabaruah** [comment]
> Everything looks fine, the is well structured, just one advice which Akshay has already mentioned above to replace `for` attribute in the label to `htmlFor`, rest all looks good.

**Comment by shristabaruah** [comment]
Yes , Thank you @AtulPant2704 I made the changes!

**Pull request merged**